README.md

```diff
@@ -10,7 +10,6 @@ description: Top-level directory for official Azure Machine Learning sample code
 
 # Azure Machine Learning examples
 
```

Comment:
Any thoughts on replacing this smoke badge with a code-quality-python one (is that supported?)?

Reply:
I don't know that the badge was really communicating anything useful to users in the first place. Knowing that `smoke` was passing didn't really mean anything to someone who didn't know what that was, and I imagine that knowing that`code-quality-python` is passing would be similarly unclear.